test/serviceprofiles/testdata/tap_application.yaml

```diff
@@ -0,0 +1,197 @@
+# slow_cooker --http-> gateway --grpc-> t1
```

Comment:
what about reusing `test/tap/testdata/tap_application.yaml`?

Reply:
I was thinking about that too, but then I thought it would be better to have a `tap_application.yaml` specifically for `serviceprofiles`. If we want to modify t* services for specific test scenarios. e.g modifying t3 to fail 50% of the time to test out and validate retries.

Reply:
Makes sense, and I see you just added `--percent-failure`... mind bumping the `bb` images to `v0.0.5` to match the original version?